mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
```

Comment:
The line 'res.status(200).json({ message: "Order deleted successfully" });' is duplicated outside the 'try' block in the 'deleteOrder' function. This is unnecessary and could lead to unexpected behavior. It should be removed to ensure the response is sent only once after the order deletion process is completed successfully.